README.md

```diff
@@ -1,2 +1,61 @@
-# mobile-poc
-Poc mobile of dis moi
+# DisMoi - mobile
+> _Dis moi_ means _Tell me_ in :fr: French.
+
+**Dismoi** is a web extension that allows anyone to post information directly on any web page you browse.
```

Comment:
So DisMoi mobile is the same as the extension?

Reply:
Haha no... mistake

Reply:
Oh actually @lutangar I've just copied paste from the other repos. It's a general message explaining what is DisMoi. Maybe the main message describing what is dismoi should change?

Reply:
Actually what I would expect here is what's this PoC is about? What are we trying to achieve here? Which concept are we trying to prove?
You should propose something with your own words, anything will do, but something specific to this project.

Reply:
Agreed